docs/tigerbeetle.md

```diff
@@ -0,0 +1,19 @@
+# Rafiki and Tigerbeetle
+
+TigerBeetle is a distributed financial accounting database designed for mission critical safety and performance. Since Rafiki implements the [Interledger Protocol](./glossary.md#interledger-protocol), which splits payments into packets, we need a high performing and safe database to store all that payment data. For detailed information on Tigerbeetle, including its consesus mechanism and its limitations, visit the official Tigerbeetle [documentation](https://docs.tigerbeetle.com/) and [blog](https://tigerbeetle.com/blog/).
+
+Rafiki uses a combination of liquidity and settlement accounts to perform double-entry accounting. These accounts correspond to Tigerbeetle credit and debit accounts, respectively. For more information on Rafiki's accounting, refer to [Accounts and Transfers](./accounts-and-transfers.md). Note that Tigerbeetle only holds balance data an no additional metadata included in ILP packets. Such metadata is stored in a [Redis](https://redis.io/) database.
```

Comment:
I'm not sure that the part about metadata being stored in Redis is correct. @wilsonianb I'm sure you know more about that.

Reply:
Redis is only used for tracking a stream's total received amount (which might only be used for STREAM receipts)
https://github.com/interledger/rafiki/blob/8c0967bea1294318cc8c5a9f11b4fcc1933800e2/packages/backend/src/connector/core/controllers/stream.ts#L36-L47

Reply:
So are we storing any additional metadata from the packets? (This is why I want you to write the connector docs @wilsonianb 😉)